ocaml/lambda/translcore.ml

```diff
@@ -30,8 +29,8 @@ open Debuginfo.Scoped_location
 type error =
     Free_super_var
   | Unreachable_reached
-  | Bad_probe_layout of Ident.t
-  | Illegal_record_field of Sort.const
+  | Bad_probe_jkind of Ident.t
```

Comment:
Another error I think is really about layouts.  (That is, I think we wouldn't have trouble compiling probes that capture things where the other bits of the jkind are unusual).

Reply:
OK